dapp/src/insurance/testflightdelay.sol

```diff
@@ -0,0 +1,745 @@
+/*
+
+	FlightDelay with Oraclized Underwriting and Payout
+	All times are UTC.
+	Copyright (C) Christoph Mussenbrock, Stephan Karpischek
+	
+*/
```

Comment:
Seems to have been copy pasted from an other repo, better to put a link to that document in dapp/src/insurance/readme.md